Sources/WordPressUI/Extensions/BlockEvents/UIBarButtonItem+BlockEvents.swift

```diff
@@ -25,8 +25,16 @@ extension UIBarButtonItem: ControlEventBindable { }
 // MARK: - Implementation
 extension ControlEventBindable where Self: UIBarButtonItem {
     private var controlEventHandlers: [BarButtonItemEventHandler<Self>] {
-        get { return (objc_getAssociatedObject(self, &BlockEventKeys.ControlEventHandlers) as? [BarButtonItemEventHandler<Self>]) ?? [] }
-        set { objc_setAssociatedObject(self, &BlockEventKeys.ControlEventHandlers, newValue, .OBJC_ASSOCIATION_RETAIN_NONATOMIC) }
```

Comment:
Changing `ControlEventHandlers` to be an Int (i.e. `ControlEventHandlers = 0`) would get rid of the warning.

Reply:
I've quite literally done that exact same thing [a few days ago](https://github.com/bloom/DayOne-Apple/blob/trunk/core/DOCore/DOCore/Sync/URLSessionTask%2BMedia.swift#L43)!

(In my scenario, I'm not quite sure why, but the compiler *wanted* a `UInt8`, but just checked and `Int` should suffice)

Reply:
Addressed in dfca54a8c77045d64fa26cb2d37b26cacf3e7cf6, but I didn't use `0` for all the keys because AFAICT it's basically a hash, so each associated object needs its own unique key?

Thanks for the suggestion!!

Reply:
> it's basically a hash, so each associated object needs its own unique key?

Yes. But the hash key"is the variable's memory address. Since each variable has its own unique memory address, I don't think the variable value really matters here.

Reply:
Thank you gentleman!!